docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -556,26 +864,81 @@ Start the second and the third node on two separate VMs:
+You can set the data placement constraint manually using the `--constraint_value` flag, which takes the comma-separated value of `cloud.region.zone`. For example:
 
 ```sh
-./bin/yugabyted configure --fault_tolerance=zone --data_placement_constraint=aws.us-east.us-east-1a,aws.us-east.us-east-2a,aws.us-east.us-east-3a
+./bin/yugabyted configure data_placement --fault_tolerance=zone --constraint_value=aws.us-east.us-east-1a,aws.us-east.us-east-2a,aws.us-east.us-east-3a
 ```
 
 You can set the replication factor of the cluster manually using the `--rf` flag. For example:
 
 ```sh
-./bin/yugabyted configure --fault_tolerance=zone --data_placement_constraint=aws.us-east.us-east-1a,aws.us-east.us-east-2a,aws.us-east.us-east-3a --rf=3
+./bin/yugabyted configure data_placement --fault_tolerance=zone --constraint_value=aws.us-east.us-east-1a,aws.us-east.us-east-2a,aws.us-east.us-east-3a --rf=3
 ```
 
-### Create a multi-region cluster
+### Create a multi-zone cluster (Secure)
+
+{{< note title="Note" >}}
+
+openssl must be installed in the 1st VM to deploy a secure cluster.
+
+{{< /note >}}
+
+To create a multi-node secrue cluster, you start the first node by running the `yugabyted start` command, using the `--secure` flag and passing in the `--cloud_location` and `--fault_tolerance` flags to set the node location details, as follows:
+
+```sh
+./bin/yugabyted start --secure --advertise_address=<host-ip> --cloud_location=aws.us-east.us-east-1a --fault_tolerance=zone
+```
+
+Create the certificates for 2nd and 3rd VM for SSL/TLS connection.
+
+```sh
+./bin/yugabyted cert generate_server_certs --hostnames=<IP_of_VM_2>,<IP_of_VM_3>
+```
+
+Manually copy the generated certificates in the 1st VM to 2nd and 3rd VM.
+
+Copy the certificates for 2nd VM from `$HOME/var/generated_certs/<IP_of_VM_2>` in 1st VM to `$HOME/var/certs` in 2nd VM.
+Copy the certificates for 3nd VM from `$HOME/var/generated_certs/<IP_of_VM_3>` in 1st VM to `$HOME/var/certs` in 3nd VM.
+
+Start the second and the third node on two separate VMs:
+
+```sh
+./bin/yugabyted start --secure --advertise_address=<host-ip> --join=<ip-address-first-yugabyted-node> --cloud_location=aws.us-east.us-east-2a --fault_tolerance=zone
+
+./bin/yugabyted start --secure --advertise_address=<host-ip> --join=<ip-address-first-yugabyted-node> --cloud_location=aws.us-east.us-east-3a --fault_tolerance=zone
+```
+
+After starting the yugabyted processes on all the nodes, configure the data placement constraint of the cluster:
+
+```sh
+./bin/yugabyted configure data_placement --fault_tolerance=zone
+```
+
+This command automatically determines the data placement constraint based on the `--cloud_location` of each node in the cluster. If there are 3 or more zones available in the cluster, the configure command configures the cluster to survive at least 1 availability zone failure. Otherwise, it outputs a warning message.
```

Comment:
The preceding command automatically determines the data placement constraint based on the `--cloud_location` of each node in the cluster. If there are three or more zones available in the cluster, the `configure` command configures the cluster to survive at least one availability zone failure. Otherwise, it outputs a warning message.